lib/bolt/project_migrator.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'bolt/project_migrator/config'
+require 'bolt/project_migrator/inventory'
+require 'bolt/project_migrator/modules'
+
+module Bolt
+  class ProjectMigrator
+    def initialize(config, outputter)
+      @config    = config
+      @outputter = outputter
+    end
+
+    def migrate
+      ok = migrate_inventory && migrate_config && migrate_modules
```

Comment:
I think we only want to do `migrate_modules` if BOLT_MODULE_FEATURE is enabled, for now.